src/corehost/packaging/Microsoft.NETCore.App/project.json

```diff
@@ -0,0 +1,46 @@
+{
+  "dependencies": {
+    "Microsoft.CodeAnalysis.CSharp": "1.3.0-beta1-20160321-04",
+    "Microsoft.CodeAnalysis.VisualBasic": "1.3.0-beta1-20160321-04",
+    "Microsoft.CSharp": "4.0.1-rc2-23931",
+    "Microsoft.NETCore.DotNetHostPolicy": "1.0.1-beta-002200",
```

Comment:
The long-term vision is that this will move out of the CLI. But for RC2 @gkhanna79 and @schellap have a short-term plan on how to make it work. Can you guys elaborate the plan?